cmake/toolchain/linux_arm64.cmake

```diff
@@ -7,4 +7,9 @@ set(CMAKE_CXX_COMPILER_ID "GNU")
 set(CMAKE_AR ar CACHE STRING "ar")
 set(CMAKE_CXX_COMPILER_WORKS TRUE)
 set(CMAKE_C_COMPILER_WORKS TRUE)
-set(NO_XCB TRUE)
+set(BUILD_WSI_XCB_SUPPORT OFF CACHE BOOL "" FORCE)
+set(BUILD_WSI_WAYLAND_SUPPORT OFF CACHE BOOL "" FORCE)
+set(BUILD_WSI_XLIB_SUPPORT OFF CACHE BOOL "" FORCE)
+set(BUILD_ZSTD_SUPPORT OFF CACHE BOOL "" FORCE)
+set(BUILD_WERROR OFF CACHE BOOL "" FORCE)
+include_directories("${CMAKE_SOURCE_DIR}/external/precompiled/linux/include")
```

Comment:
Could you change this to use `target_include_directories` and only add this for the targets that need it?

Reply:
Ok.